styles/custom.css

```diff
@@ -1,10 +1,47 @@
-/* Footer styles */
+.navbar:not(.navbar-fixed-top){
+  margin-bottom: 0;
+}
+body {
+  line-height: 1.8;
+}
+
+body .navbar *,
+body footer *
+{
+  /* Reset line-height for navbar and footer */
+  line-height: 1.42857143;
+}
+
+/* Sticky footer */
+html {
+  position: relative;
+  min-height: 100%;
+}
+body {
+  /* Margin bottom by footer height */
+  margin-bottom: 60px;
+}
+body > .container {
+  /* Push footer 20px from content */
+  margin-bottom: 80px;
+}
 .footer {
-  margin-top: 50px;
-  padding: 50px 0;
   background: #222;
   color: #9d9d9d;
+
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  /* Set the fixed height of the footer here */
+  height: 60px;
```

Comment:
That line breaks footer on mobile devices, as text is to long to fit in one line and, due to `line-height` footer has at least 180px of height.